src/pages/patientView/trialMatch/TrialMatchTableUtils.ts

```diff
@@ -0,0 +1,129 @@
+import {
+    IArmMatch, IClinicalGroupMatch, IDetailedTrialMatch, IGenomicGroupMatch, IGenomicMatch, ITrial,
+    ITrialMatch, IArm, IDrug
+} from "../../../shared/model/MatchMiner";
+import * as _ from 'lodash';
+
+export function groupTrialMatchesById(trials: ITrial[], trialMatches:ITrialMatch[]): IDetailedTrialMatch[] {
+    const matchesGroupedById = _.groupBy(trialMatches, (trial: ITrialMatch) => trial.id);
+    let matchedTrials: IDetailedTrialMatch[] = [];
+    _.forEach(matchesGroupedById, (trialGroup, trialId) => {
+        const originalMatchedTrial: ITrial = _.find( trials, { 'id': trialId } )!;
```

Comment:
should be `_.find(trials, (trial)=>trial.id===trialId)`
It's typesafe that way.   We've decided to never use the lodash shortcuts.

Reply:
Updated